stable/insights-agent/templates/rbac-reporter/serviceaccount.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
I don't think we need a separate serviceaccount - we can just use the one that's created in serviceaccount.yaml